sw/device/tests/rom_ext/rom_ext_parser_unittest.cc

```diff
@@ -65,11 +65,31 @@ TEST_F(SignatureGetTest, Success) {
   EXPECT_THAT(src_.data, ElementsAreArray(dst.data));
 }
 
-class ImageLengthGetTest : public ParserTest {};
+class ExtentsGetTest : public ParserTest {
+ protected:
+  testing::Matcher<rom_ext_extents_t> Eq(const rom_ext_extents_t &rhs) {
+    return testing::AllOf(
+        testing::Field("image_start", &rom_ext_extents_t::image_start,
+                       rhs.image_start),
+        testing::Field("signed_area_start",
+                       &rom_ext_extents_t::signed_area_start,
+                       rhs.signed_area_start),
+        testing::Field("image_end", &rom_ext_extents_t::image_end,
+                       rhs.image_end));
+  }
```

Comment:
Can you give this a name that isn't `Eq`, or else implement `operator==` and `operator<<` for `rom_ext_extends_t`?

I don't want this to be confusable with `testing::Eq`.

Reply:
I've renamed this to `EqualsRanges`, but it's still a member function.